schema/release-schema.json

```
@@ -1547,7 +1547,7 @@
     "Item": {
       "title": "Item",
       "type": "object",
-      "description": "A good, service, or work to be contracted.",
+      "description": "A single instance of the goods, services, or works to be contracted.",
```

Per https://github.com/open-contracting/standard/pull/1530#issuecomment-1163380962 an item can be plural so I'm not sure 'single instance' is quite the right phrasing. Another option could be to use the wording from https://standard.open-contracting.org/latest/en/schema/reference/#item:

```suggestion
      "description": "A line-item of the goods, services, or works to be contracted.",
```

---

I was attempting to find a way to express that an item will be a single type. To take the example of "cleaning services", these can take many forms but if the classification being used is simply "cleaning services" without further disambiguation then this is being presented as a single, unique type of service. "Type" didn't feel like the right word though. "Line-item" could work but it is then a bit of a circular description, "an item is a line-item...".

---

Hmm, yes, it's a tricky one. I checked the [glossary crosswalk](https://docs.google.com/spreadsheets/d/1Nh_HjyYuNk8wuu_Tb0kcPDyV6j8pAMgXqJ3MgwpXZjQ/edit?skip_itp2_check=true#gid=0) and the item concept lacks a definition. I also looked at a few definitions of 'line item' ([Collins](https://www.collinsdictionary.com/dictionary/english/line-item), [Wiktionary](https://en.wiktionary.org/wiki/line_item), [Law Insider](https://www.lawinsider.com/dictionary/purchase-order-line)), which are quite circular, too.

How about:

```suggestion
      "description": "A specific instance of the goods, services, or works to be contracted.",
```
That works for the examples given in https://github.com/open-contracting/standard/pull/1530#issuecomment-1163380962